tasks.py

```diff
@@ -153,60 +255,113 @@ def post_upgrade(context):
     - clearsessions
     - invalidate all
     """
-    docker_compose(context, "run nautobot nautobot-server post_upgrade")
+    command = "nautobot-server post_upgrade"
+
+    if local is None:
+        local = context.nautobot.local
+    if is_truthy(local):
+        context.run(command)
+    docker_compose(context, f"run nautobot {command}")
```

Comment:
Mind adding a small helper function to eliminate this five-line repeated boilerplate?